### DISCUSSION TITLE
New Installation: Only Headings Appear

### DISCUSSION THREAD

**Discussion by MundoMedia** [FreshRSS/FreshRSS#6742] [Q&A]
Hello,
I just installed Fresh RSS on my server, it all seemed to install fine (all 'ticks' when it was done). But I notice that for the feeds I've added, only the headings / Article Titles appear.
I can't seem to find any setting that would allow descriptions / meta descriptions / text / etc to appear.

Is it possible to have anything more than the titles/headings appear in the reader?

**Comment by Alkarex** [comment]
Please include a screenshot and information about your system / setup.

**Comment by MundoMedia** [comment]
Thanks.
Here are some screenshots, although I'm not sure what would be relevant -- let me know if you need a specific image.
I've installed FreshRSS in a subdirectory of my Public directory -- where a Wordpress installation is also hosted.
It is on a shared account at Racknerd

https://www.racknerd.com/shared-hosting

For installation I downloaded the latest zip from:
https://github.com/FreshRSS/FreshRSS?tab=readme-ov-file#installation

uploaded it to the subdirectory and extracted, and made the necessary directories public as the instructions suggest.
The instructions were pretty simple and clear, and no errors displayed when It finished, so I figured it all went ok.

![01_69786355-3768-4f77-8d4f-53f490965937](https://github.com/user-attachments/assets/c27eeed5-6e7c-4fe2-acc5-1706d6d1bad0)
![02_e88c13aa-c06d-4eee-829d-2da6a0598ba2](https://github.com/user-attachments/assets/5858f9f0-7175-4923-a9f3-ad984749a61c)
![03_28afbfd6-1ca6-48ad-98a0-a103a89d5f2d](https://github.com/user-attachments/assets/21d06266-8505-4c70-873f-ccc92e29335b)

**Comment by Alkarex** [comment]
This looks correct and it is probably due to the RSS feed you have chosen not having any content (what is its address?).
How do other feeds like `https://freshrss.org/feeds/all.atom.xml` look like?

**Comment by MundoMedia** [comment]
Thanks for replying again.

So, the 2 feeds I added (only just installed, so I was just testing) are:
https://news.mit.edu
= https://news.mit.edu/topic/mitmachine-learning-rss.xml
and
https://www.nature.com/subjects/machine-learning
= https://www.nature.com/subjects/machine-learning.atom

The MIT one seems to contain the full content. But, yes, the Nature one doesn't :(

Actually, I only pasted in the urls and FreshRSS returned the feed urls.

So, I was searching around on this issue and found this article:
`https://joelchrono.xyz/blog/fetch-full-article-content-freshrss/`

So, I followed what he writes and by adding the ,article> element tag for the MIT site, it now provides the full text and images. Which is great.

So, maybe I've solved the 'problem'. But the guy who wrote the article seems to think this is an 'obscure' way of doing it and a bit of a 'hack' so I am not sure if it is the 'right' way to do it.

I guess the 'advanced' settings are there for a reason, but is it necessary to search for html tags / css classes/ids for every site to get full content or is there a better way?

I also found fivefilters.org which seems to be some kind of add-on, but that's a paid service and I'm reluctant to pay for a service for a free to use app! (Seems contradictory!).

So, anything you might advise or suggest, I'd be grateful to hear. Thanks again for getting back and a great RSS reader! -- Easiest installation ever!
